### ISSUE TITLE
Flappy bird Minigame controls improvement

### ISSUE THREAD

**Issue by Nicodaw** [Nicodaw/Uni-UniversityDominationA4#3]
Current implementation
---------------------------
Game starts after a 3 second countdown. Flight curve too steep.

Desired functionality
----------------------
Improve general feel of the controls. Suggestion is make the time the bird is at peak flight slightly longer.
_Optional if there is time left:_ replace the 3 second countdown to start the game with just [flapping once](http://flappybird.io/)

**Comment by Gorea235** [comment]
Changing the game intro should be easy, since it would be a case of swapping out the timer for an input based system. The flight curve I think is just a case of gravity, if we make the gravity weaker the curve should flatten out more

**Comment by Gorea235** [comment]
As of 91044a0, the minigame has been rewritten to play much more nicely. I've cleaned up the prefabs and scene to make it more simple internally, and I have increased the reliance on Unity for things like gravity. The jump curve has been drastically improved, and the pillars & clouds have also been tweaked, meaning the game should be much nicer to play. I've also moved to the more flappy bird-like system of only starting when the user give the first input, meaning people should be able to start it much better.
